src/common/showToast.js

```diff
@@ -0,0 +1,3 @@
+import Toast from 'react-native-simple-toast';
```

Comment:
This likely belongs in utils.
If you are going to have more functions related to toast call it 'toats.js' if not export default.